internal/blueprint/customizations_test.go

```diff
@@ -0,0 +1,158 @@
+package blueprint
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestCustomizationsHostname(t *testing.T) {
```

Comment:
I suggest using the name of the function under test as part of the name of the test func, e.g. `TestCustomizationsGetHostName` or even `TestGetHostName` b/c the filename is `blueprins/customizations_test.go` which already points to `blueprints/customizations.go` as the general test subject.